config/config_tests.xml

```diff
@@ -348,6 +348,18 @@ NODEFAIL          Tests restart upon detected node failure. Generates fake failu
     <DOUT_S>FALSE</DOUT_S>
   </test>
 
+  <test NAME="LDSTA">
+    <DESC>Tests the short term archiver's last date functionality.</DESC>
+    <INFO_DBUG>1</INFO_DBUG>
+    <STOP_OPTION>nmonths</STOP_OPTION>
```

Comment:
24 months is way too long in my opinion.  The test should be able to be run in just a few timesteps or at worst a few days.

Reply:
24 months is very long, yes, but I need to test the functionality for different years. I think better would be to get a mock run file generator going, and not use this as a system test, and just a scripts_regression_tests test again

Reply:
What about just setting the start date to Dec 30th and running a few days?  How would a mock run file generator predict the run files from the various components?  Using config_archive.xml to anticipate file names perhaps?

Reply:
Making a mock file generator work correctly is one of my main concerns, I haven't put enough thought into how it would work. It's complicated by components (MPAS...) using non-standard file names.
Setting the start date to Dec 30th seems like the best immediate solution, thanks for pointing it out.